dev-resources/keycloak_demo/test-realm.json

```diff
@@ -1252,6 +1252,15 @@
       "display.on.consent.screen" : "true",
       "consent.screen.text" : "xAPI Read Only"
     }
+  }, {
+    "id" : "0448c583-b1f9-4cc5-90fd-b7c591966e69",
+    "name" : "lrs:statements/read/mine",
+    "description" : "Read own statements",
+    "protocol" : "openid-connect",
+    "attributes" : {
+      "include.in.token.scope" : "true",
+      "display.on.consent.screen" : "true"
+    }
```

Comment:
I see that this scope was added in addition to `lrs:all/read`. Wouldn't `lrs:all/read` overwrite the new scope though?

Reply:
this is the client scope itself. What applies is the `defaultClientScopes` of the client, `lrs`

Reply:
Wouldn't you also need `lrs:statements/write` since you added that to the default scopes?

Reply:
it's in there https://github.com/yetanalytics/lrsql/blob/37c779e7ac0cf00f81a3c40794db58764a995574/dev-resources/keycloak_demo/test-realm.json#L1157-L1167